EIPS/eip-4521.md

```diff
@@ -0,0 +1,62 @@
+---
+eip: 4521
+title: 721/20-compatible transfer
+description: Recommends a simple extension to make NFTs compatible with apps and contracts that handle fungibles.
+author: Ross Campbell (@z0r0z)
+discussions-to: https://github.com/ethereum/EIPs/issues/4521
```

Comment:
Your `discussions-to` is 404ing. Can you create a topic on https://ethereum-magicians.org please?

Reply:
yes @lightclient I have made a thread here: https://ethereum-magicians.org/t/eip-4521-721-20-compatible-transfer/7903